<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>angleFilterWdg</class>
 <widget class="QWidget" name="angleFilterWdg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>434</width>
    <height>498</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>angleFilterWdg</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="bt_select">
         <property name="text">
          <string>选直线</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="bt_ensure">
         <property name="text">
          <string>确定</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="bt_cancel">
         <property name="text">
          <string>取消</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>313</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="2">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>73</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QCheckBox" name="cb_matrixRefer">
         <property name="text">
          <string>旋转矩阵</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cb_dispAfterDet">
         <property name="text">
          <string>测后显示</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0" colspan="3">
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>参数选择</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QComboBox" name="cb_lineIndex">
          <item>
           <property name="text">
            <string>基准区域</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>左管脚正向</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>右管脚正向</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>左管脚反向</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>右管脚反向</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="bt_allUes0Param">
          <property name="text">
           <string>所有区域使用基准区域参数</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="3">
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>线属性</string>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="3">
            <widget class="QSpinBox" name="sp_step">
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>10</number>
             </property>
            </widget>
           </item>
           <item row="2" column="3">
            <widget class="QSpinBox" name="sp_rmDist">
             <property name="minimum">
              <number>1</number>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QPushButton" name="bt_advance">
             <property name="text">
              <string>高级参数</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QLabel" name="lb_iteratNum">
             <property name="text">
              <string>最大迭代次数</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QSpinBox" name="sp_gradient">
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>255</number>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QSpinBox" name="sp_degRange">
             <property name="minimum">
              <number>1</number>
             </property>
            </widget>
           </item>
           <item row="1" column="3">
            <widget class="QSpinBox" name="sp_iteratNum">
             <property name="minimum">
              <number>1</number>
             </property>
            </widget>
           </item>
           <item row="6" column="2">
            <widget class="QLabel" name="lb_pol">
             <property name="text">
              <string>极性</string>
             </property>
            </widget>
           </item>
           <item row="5" column="3">
            <widget class="QComboBox" name="cb_first">
             <item>
              <property name="text">
               <string>最大对比</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>第一条</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="6" column="3">
            <widget class="QComboBox" name="cb_pol">
             <item>
              <property name="text">
               <string>双向</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>黑到白</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>白到黑</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>对比度</string>
             </property>
            </widget>
           </item>
           <item row="5" column="2">
            <widget class="QLabel" name="lb_first">
             <property name="text">
              <string>直线</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QLabel" name="lb_step">
             <property name="text">
              <string>毛边</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>角度范围</string>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QLabel" name="lb_rmDist">
             <property name="text">
              <string>排除距离</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="2" colspan="2">
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>182</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="3">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item row="2" column="0" colspan="3">
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>角度范围</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="3">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="2">
         <widget class="QSpinBox" name="sp_maxAng1">
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::UpDownArrows</enum>
          </property>
          <property name="maximum">
           <number>360</number>
          </property>
          <property name="value">
           <number>360</number>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="sp_minAng1">
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::UpDownArrows</enum>
          </property>
          <property name="maximum">
           <number>360</number>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSpinBox" name="sp_minAng0">
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::UpDownArrows</enum>
          </property>
          <property name="maximum">
           <number>360</number>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QSpinBox" name="sp_maxAng0">
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::UpDownArrows</enum>
          </property>
          <property name="maximum">
           <number>360</number>
          </property>
          <property name="value">
           <number>360</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>基准区域-右管脚</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>基准区域-左管脚</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <spacer name="horizontalSpacer_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>22</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>1</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
